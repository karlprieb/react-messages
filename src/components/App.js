import React from 'react'
import Counter from '../containers/Counter'
import ChatBox from '../containers/ChatBox'
import AddMessage from '../containers/AddMessage'

import './app.css'

const App = () => (
  <main className='main-wrapper'>
    <Counter />
    <ChatBox />
    <AddMessage />
  </main>
)

export default App
