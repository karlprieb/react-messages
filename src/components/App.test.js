import React from 'react'
import App from './App'

describe('App component', () => {
  it('should render all chat application', () => {
    const wrapper = shallow(<App />)
    expect(wrapper).toMatchSnapshot()
  })
})
