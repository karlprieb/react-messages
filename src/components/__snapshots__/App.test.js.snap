// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render all chat application 1`] = `
<main
  className="main-wrapper"
>
  <Connect(MessageCounter) />
  <Connect(MessageList) />
  <Connect(MessageField) />
</main>
`;
